docs/JAYDEN.md

```diff
@@ -1,40 +1,74 @@
-<!-- 
+<!--
  @requires
  1. VSCode extension: Markdown Preview Enhanced
  2. Shortcut: 'Ctrl' + 'Shift' + 'V'
  3. Split: Drag to right (->)
 
+
  @requires
  1. VSCode extension: Markdown All in One
  2. `File` > `Preferences` > `Keyboard Shortcuts`
  3. toggle code span > `Ctrl + '`
  4. toggle code block > `Ctrl + Shift + '`
 
+
  @usage
  1. End of Proof (Q.E.D.): <div style="text-align: right;">&#11035;</div>
- 2. End of Each Section: 
+ 2. End of Each Section:
+
 
      <br /><br /><br />
 
+
      ---
 
 
 
+
+
+
      <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+
  3. ![image_title_](images/imagefile.png)
  4. [url_title](URL)
  -->
 <!-- Anchor Tag (Object) for "back to top" -->
-<a id="readme-top"></a> 
+<a id="readme-top"></a>
+
+
+
 
 
 
 ## Table of Contents
 - [Math: SAT](#math-sat)
+- [📐 Triangle Inequality Theorem](#-triangle-inequality-theorem)
+  - [🎯 What is the Triangle Inequality Theorem?](#-what-is-the-triangle-inequality-theorem)
```

Comment:
For headings, use short, but descriptive nouns, not sentences.

Headers are supposed to be less then 3 words, sometimes with parentheses. ()